index.html

```diff
@@ -70,6 +71,9 @@ <h1>Yash</h1>
 <!-- Countdown Timer -->
 <div id="time" style="text-align: center; margin-top: 20px;"></div>
 
+<div id="wedding" class="wrap">
+   <button class="gift-btn" id="gift-btn">Gift Now&nbsp;&nbsp;&nbsp;</button>
```

Comment:
the style element seems to be missing

Reply:
![Uploading Screenshot 2024-10-23 at 8.47.48 AM.png…]()

Reply:
raised this PR; https://github.com/shivikawedsyash/invitation/pull/12